pyproject.toml

```diff
@@ -27,6 +27,7 @@ jinja2 =  "*"
 pyrdfj2 =  "*"
 furo = "*"
 recommonmark = "*"
+pandas = "*"
```

Comment:
Should we have a convention stating that we limit ourselves to the latest mayor version of a package eg: 4.x.x ? This to prevent later failures with other library implementations.